### PR TITLE
chore: add fastlane metadata for f-droid

### PR DESCRIPTION
**Pull request by thecockatiel**
related to #9210

### CODE REVIEW

**Comment by SomberNight** [comment]
Could you give some context: why is this called "fastlane"? Did you generate these files (the file hierarchy at least) using the [similarly named tool](https://github.com/fastlane/fastlane)?

Have you downloaded these from the google play store?
(maybe we should update that icon there, it's one of the worse looking ones :P)

Maybe this should go into `contrib/android/`.

**Comment by thecockatiel** [comment]
Hi
Sure,
this is the structure f-droid expects the repo to have: https://f-droid.org/docs/All_About_Descriptions_Graphics_and_Screenshots/#in-the-applications-source-repository

we can put it also in `metadata/` instead of `fastlane/...`

**Comment by thecockatiel** [comment]
and no i did not use a tool to generate these, I made them by hand
and yes i did download the images from google play store

sure thing, you can adjust the metadata as you see fit, its just a starting point for f-droid to pickup the metadata

File: fastlane/metadata/android/en-US/full_description.txt
Comment (SomberNight) [inline fastlane/metadata/android/en-US/full_description.txt:1]: I guess we also need a `short_description.txt` (?)
[Your link](https://f-droid.org/docs/All_About_Descriptions_Graphics_and_Screenshots/#in-the-applications-source-repository) says it is mandatory.

**Comment by SomberNight** [comment]
> we can put it also in metadata/ instead of fastlane/..., but we cant put it in contrib/android/

OK. Then let's keep it as-is.

File: fastlane/metadata/android/en-US/full_description.txt
Comment (thecockatiel) [reply fastlane/metadata/android/en-US/full_description.txt:1]: oops
I did not see a short desc on google play thats why I omitted it
I'll add one

Comment (thecockatiel) [reply fastlane/metadata/android/en-US/full_description.txt:1]: done

Comment (SomberNight) [reply fastlane/metadata/android/en-US/full_description.txt:1]: Same link says it needs to be below 80 chars, so yours is too long.

> `│   ├── short_description.txt   (short description, max 80 chars, mandatory)`

AFAICT on the play store we have:
> `Fast and self-custodial wallet for Bitcoin and the Lightning Network`

It is visible when you search, before opening the page for any specific app:
https://web.archive.org/web/20240926162441/https://play.google.com/store/search?q=org.electrum.electrum&c=apps

Comment (thecockatiel) [reply fastlane/metadata/android/en-US/full_description.txt:1]: sorry for the mistakes and thanks for the help
I've updated the short description with the given text

Comment (SomberNight) [reply fastlane/metadata/android/en-US/full_description.txt:1]: No problem. Thanks for working on this!

**Comment by thecockatiel** [comment]
when will this PR get merged ? just asking because this has to be inside the repo before the MR on f-droid

**Comment by thecockatiel** [comment]
thanks

**Pull request merged**